docs/source/markdown/podman-machine-init.1.md.in

```diff
@@ -39,6 +39,14 @@ Size of the disk for the guest VM in GB.
 
 Print usage statement.
 
+#### **--identity**
+
+Relative or absolute path to use for identity files (private key part). If the
```

Comment:
```suggestion
Relative or absolute path for identity files (private key part). If the
path is relative, it will be evaluated relative to the default .ssh
directory. All directories should be created in advance. If this is unset, whitespace, or empty, then the filename would be dynamically chosen for the
machine.
```